packages/tadviewer/src/components/AppPane.tsx

```diff
@@ -60,6 +60,19 @@ export interface AppPaneBaseProps {
     exportPath: string,
     parquetExportOptions: ParquetExportOptions
   ) => void;
+  onCellClick?: (cell: AugmentedCellData ) => void;
+}
+
+export type CellData = {
+  value: string;
+  columnName: string;
+  cell: { row: number; col: number };
+};
+
+export type AugmentedCellData = {
+  value: string;
+  cell: { row: number; col: number };
+  column: { displayName: string; columnType: string };
```

Comment:
A few suggestions about these types:
- Consider flattening these types, and making one inherit from the other (adding `columnType`).
- Also consider adding `readonly` to the properties.
- Prefer using `interface` instead of `type`. They're almost identical, but the former can result in better IDE feedback for types, so is preferable when possible (as it is here).
- Consider using the appropriate one in TadViewerPane, instead of duplicating the definition, and also exporting it (see my comment in your other PR about the duplication there).
- Naming: perhaps `CellClickDataWithoutColumnType` and `CellClickData`? Normally I'd suggest `CellClickData` and `CellClickDataWithColumnType`, but the latter is the exported one, so giving it the shorter name is better.

Reply:
Was able to reduce the type to one type by moving the augmentation down a level, per your suggestion.
Updated to readonly/interface, and exporting from TadViewerPane, and going with the single type, `CellClickData`. Thanks!